packages/web/index.html

```diff
@@ -29,7 +29,7 @@
     </script>
   </head>
   <body>
-    <div id="root"></div>
+    <div id="root" class="flex flex-col"></div>
```

Comment:
Agree with most of these changes, but I prefer we have 0 styling in this file. It makes for a cleaner root and need to open less files when "looking for the rule".

I would add this back to the Layout